pallets/loans-ref/src/pricing/external.rs

```diff
@@ -0,0 +1,110 @@
+use cfg_primitives::Moment;
+use cfg_traits::{
+	data::{DataCollection, DataRegistry},
+	ops::{EnsureAddAssign, EnsureDiv, EnsureMul, EnsureSub, EnsureSubAssign},
+};
+use cfg_types::adjustments::Adjustment;
+use codec::{Decode, Encode, MaxEncodedLen};
+use frame_support::{self, ensure, RuntimeDebugNoBound};
+use scale_info::TypeInfo;
+use sp_runtime::{traits::Zero, DispatchError, DispatchResult};
+
+use crate::pallet::{Config, Error, PoolIdOf, PriceOf};
+
+/// External pricing method
+#[derive(Encode, Decode, Clone, PartialEq, Eq, TypeInfo, RuntimeDebugNoBound, MaxEncodedLen)]
+#[scale_info(skip_type_params(T))]
+pub struct ExternalPricing<T: Config> {
+	/// Id of an external price
+	pub price_id: T::PriceId,
+
+	/// Maximum number of items associated to the price id
```

Comment:
Couldn't a `PriceId` be reused by multiple loans? In that case, wouldn't this rather be "Maximum number of items associated with the loan of the pricing?" or "Initial number of items associated with the loan of the pricing"

Reply:
> Couldn't a PriceId be reused by multiple loans?

Yes! I like the first suggestion 👍🏻